controllers/envoy/envoyconfigrevision_controller.go

```diff
@@ -101,20 +102,21 @@ func (r *EnvoyConfigRevisionReconciler) Reconcile(req ctrl.Request) (ctrl.Result
 			return ctrl.Result{}, nil
 		}
 
-		// Push the snapshot to the xds server cache
-		oldSnap, _ := (*r.ADSCache).GetSnapshot(nodeID)
-		if !envoy.ResourcesEqual(snap.Resources, (&oldSnap).Resources) {
-			// TODO: check snapshot consistency using "snap.Consistent()". Consider also validating
-			// consistency between clusters and listeners, as this is not done by snap.Consistent()
-			// because listeners and clusters are not requested by name by the envoy gateways
-
-			// Publish the in-memory cache to the envoy control-plane
```

Comment:
This was a remnant of an older version of marin3r which relied on checking the resources one by one to detect changes in the resources published by the discovery service server. This is now not necessary as the field `spec.version` of and `EnvoyConfigRevision` holds the hash of the resources, and can easily checked against the version of the resources in the cache (which is also the hash of the published resources).